libraries/AP_BattMonitor/AP_BattMonitor_Params.cpp

```diff
@@ -83,6 +89,57 @@ const AP_Param::GroupInfo AP_BattMonitor_Params::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("FS_VOLTSRC", 11, AP_BattMonitor_Params, _failsafe_voltage_source, BattMonitor_LowVoltageSource_Raw),
 
+    // @Param: LOW_VOLT
+    // @DisplayName: Battery low voltage
+    // @Description: Battery voltage to trigger a low battery failsafe. Set to 0 to disable. If the battery voltage drops below this voltage continuously for 10 seconds then the vehicle will switch to RTL mode.
+    // @Units: V
+    // @Increment: 0.1
+    // @User: Standard
+    AP_GROUPINFO("LOW_VOLT", 12, AP_BattMonitor_Params, _low_voltage, DEFAULT_LOW_BATTERY_VOLTAGE),
+
+    // @Param: LOW_MAH
+    // @DisplayName: Battery low capacity
+    // @Description: Battery capacity remaining to trigger a low battery failsafe. Set to 0 to disable battery remaining failsafe. If the battery capacity drops below this level the vehicle will switch to RTL mode.
+    // @Units: mA.h
+    // @Increment: 50
+    // @User: Standard
+    AP_GROUPINFO("LOW_MAH", 13, AP_BattMonitor_Params, _low_capacity, 0),
+
+    // @Param: CRT_VOLT
+    // @DisplayName: Battery critical voltage
+    // @Description: Battery voltage to trigger a criticial battery failsafe. Set to 0 to disable. If the battery voltage drops below this voltage continuously for 10 seconds then the vehicle will switch to RTL mode.
+    // @Units: V
+    // @Increment: 0.1
+    // @User: Standard
+    AP_GROUPINFO("CRT_VOLT", 14, AP_BattMonitor_Params, _critical_voltage, 0),
+
+    // @Param: CRT_MAH
+    // @DisplayName: Battery critical capacity
+    // @Description: Battery capacity remaining to trigger critical failsafe. Set to 0 to disable the critical battery remaining failsafe. If the battery capacity drops below this level the vehicle will switch to RTL mode.
+    // @Units: mA.h
+    // @Increment: 50
+    // @User: Standard
+    AP_GROUPINFO("CRT_MAH", 15, AP_BattMonitor_Params, _critical_capacity, 0),
+
+    // @Param: FS_LOW_ACT
+    // @DisplayName: Low battery failsafe action
+    // @Description: What action the vehicle should preform if it hits a low battery failsafe
```

Comment:
The action that the vehicle should perform ...

(note too perform, not preform)